modules/monitoring/files/usr/local/bin/govuk_uptime_collector

```diff
@@ -9,10 +9,11 @@ require "time"
 require "statsd"
 
 class Collector
-  def initialize(service_name, environment)
+  def initialize(service_name, environment, aws=false)
```

Comment:
Given `false` is set by default in https://github.com/alphagov/govuk-puppet/pull/6823/files#diff-ef3bfb7435b4b29de7e6bf80fd302bd3R16, we don't need to set this `=false` here, do we?

Reply:
Yep, that's true. In fact probably worth removing it.

Reply:
My first attempt at doing this was before I realised I had to edit the Puppet, so it's a hangover from that.